controller/cache/info.go

```diff
@@ -170,6 +191,45 @@ func populateIngressInfo(un *unstructured.Unstructured, res *ResourceInfo) {
 	res.NetworkingInfo = &v1alpha1.ResourceNetworkingInfo{TargetRefs: targets, Ingress: ingress, ExternalURLs: urls}
 }
 
+func populateRouteInfo(un *unstructured.Unstructured, res *ResourceInfo) {
+	route := getRoute(un)
+	targetsMap := make(map[v1alpha1.ResourceRef]bool)
+
+	if backend, ok, err := unstructured.NestedMap(un.Object, "spec", "to"); ok && err == nil {
+		targetsMap[v1alpha1.ResourceRef{
+			Kind:      kube.ServiceKind,
+			Namespace: un.GetNamespace(),
+			Name:      fmt.Sprintf("%s", backend["name"]),
+		}] = true
+	}
```

Comment:
There is only one `to` backend service per route but there can be a list of `alternateBackends`.  Please see https://docs.openshift.com/container-platform/3.6/architecture/networking/routes.html#alternateBackends